tests/sshAgent.Tests.ps1

```diff
@@ -92,7 +92,7 @@ Describe "[$global:AGENT_IMAGE] checking image metadata" {
 
 Describe "[$global:AGENT_IMAGE] image has correct version of java installed and in the PATH" {
     BeforeAll {
-        docker run --detach --tty --name="$global:CONTAINERNAME" --publish "$global:AGENT_IMAGE" $global:CONTAINERSHELL
+        docker run --detach --tty --name="$global:CONTAINERNAME" --publish-all "$global:AGENT_IMAGE" $global:CONTAINERSHELL
```

Comment:
```suggestion
        docker run --detach --name="$global:CONTAINERNAME" --publish-all "$global:AGENT_IMAGE" $global:CONTAINERSHELL
```

I'm not convinced that this change will solve all problems, but removing the `--tty` when the container is executed in background is a good practice (unless an exotic case is done such as running `cat` command on a linux container in background), we don't use TTY for background containers